src/www/ui/admin-upload-edit.php

```diff
@@ -118,7 +117,6 @@ public function Output()
       return "<h2>$text</h2>";
     }
 
-    $rc = $this->UpdateUploadProperties($upload_pk, $NewName, $NewDesc);
```

Comment:
seems this line is removed  : $rc = $this->UpdateUploadProperties($upload_pk, $NewName, $NewDesc); 
buy why?